webpack.demo.js

```diff
@@ -0,0 +1,54 @@
+var webpack = require('webpack');
+var HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: {
+    'angular2': [
+      'zone.js',
+      'reflect-metadata',
```

Comment:
So the .48 thing. We are stuck on .46 just yet so there is nothing to be done as today.

About putting extra script tags. I don't like that. That is why we have webpack at the end of the day so we can bundle and minify the deps. Unbundled for dev, bundled for prod. If we stick to script tags, we will have always dev ones or prod ones.

At the end you get the same behavior in both ways but I prefer to define my deps in webpack (lets use the tool we have in hand, right?).

That being said, I have no idea how to make that work with .48 external dependencies and webpack, but I will investigate for sure.

TL;DR I am still fine with this way and I will update to .49 when it gets out.